packages/flame_console/lib/src/commands/pause_command.dart

```diff
@@ -0,0 +1,21 @@
+import 'package:args/args.dart';
+import 'package:flame/game.dart';
+import 'package:flame_console/flame_console.dart';
+
+class PauseConsoleCommand<G extends FlameGame> extends ConsoleCommand<G> {
+  @override
+  (String?, String) execute(G game, ArgResults results) {
+    if (game.paused) {
+      return ('Game is already paused', '');
```

Comment:
Maybe say that `resume` can be used to resume it?

Reply:
I meant that the message just should say that the `resume` command can be used to start the game again, or what did you mean by "keep the command with the same name"? :thinking:

Reply:
ah right, I will change the message, I thought your meant to make the command have a toggle behaviour, which then I thought it would need to be named differently